tests/Graph/KShortestPathsFinder.cpp

```diff
@@ -322,6 +322,7 @@ TEST_CASE("KShortestPathsFinder", "[graph]") {
                   << vert.slice().get(StaticStrings::KeyString).toString();
         res = false;
       }
+      vert.destroy();
```

Comment:
This is okay for test code.

In Production please use:

AqlValueGuard guard(vert, true);
right below the line where vert is created.
This will make sure that as soon as we leave the scope (also in error cases) vert is freed.
Otherwise we would leak memory.

Reply:
done